app/controllers/api/json/imports_controller.rb

```diff
@@ -83,6 +83,42 @@ def create
     end
   end
 
+  # -------- Cancel a running import (using update PUT request) -------
+
+  def update
```

Comment:
I'd originally set this up to simply mark the job cancelled, but keep it in the database. Now that it does remove the import job as well, it would make sense to be a `DELETE`.